app/assets/javascripts/admin/line_items/controllers/line_items_controller.js.coffee

```diff
@@ -219,7 +219,7 @@ angular.module("admin.lineItems").controller 'LineItemsCtrl', ($scope, $timeout,
 
   $scope.getGroupBySizeFormattedValueWithUnitName = (value, unitsProduct, unitsVariant) ->
     scale = $scope.getScale(unitsProduct, unitsVariant)
-    if scale
+    if scale && value
       value = value / scale if scale != 28.35 && scale != 1 && scale != 453.6 # divide by scale if not smallest unit
```

Comment:
I'm curious if you can explain the magic numbers 28.35, 1 and 453.6? I'm not familiar with this code and at first glance it looks strange.

Reply:
Yep, for sure. 
This part of the app is a bit tricky, and hard to enter ... 
This values are the one that allows us to go from metric system to imperial one (or the opposite)

28.35g is 1oz
453.6g is 1lb

Reply:
Actually this could/should be constants into the app

Reply:
Thanks! Perhaps it could be worth abstracting to a service class or even a model, but I will hold that thought...

Reply:
Yeap for sure. This will help I guess.